src/website/src/pages/ec/components/forms/text-area/demo/index.js

```diff
@@ -1,4 +1,4 @@
-import demoContentDefault from '@ecl/specs-component-text-area/demo/data';
+import demoContentDefault from '@ecl/specs-component-text-area/demo/data--ec';
```

Comment:
Looks like we have to change something here, now we have default which is optional and then a duplicated example with the same optional textarea.